koku/api/migrations/0009_providerstatus_squashed_0042_auto_20200116_2048.py

```diff
@@ -34,6 +34,7 @@ def load_openshift_metric_map(apps, schema_editor):
     data = json.loads(data)
 
     for entry in data:
+        del entry["default_cost_type"]
```

Comment:
Please add a comment here about why the data is being deleted during this migration. This is something that could be easily missed if migrations are super-squashed as they were last time.

Reply:
Got it